src/d_iwad.c

```diff
@@ -58,23 +58,15 @@ static const char *const gamemode_str[] = {
     "Unknown mode"
 };
 
-// "128 IWAD search directories should be enough for anybody".
-
-#define MAX_IWAD_DIRS 128
-
 // Array of locations to search for IWAD files
+#define M_ARRAY_INIT_CAPACITY 32
```

Comment:
Does this suffice on Windows? I am currently at 15 entries on Linux.

Reply:
Yes, that should be fine.